flake.nix

```diff
@@ -0,0 +1,31 @@
+{
+  description = "An example NixOS configuration";
+
+  inputs = {
+    nixpkgs = { url = "github:nixos/nixpkgs/nixos-unstable"; };
+    nixpkgs-github-runner = { url = "github:codedownio/nixpkgs/multiple-github-runners"; };
+  };
+
+  outputs = inputs: {
+    nixosConfigurations = rec {
+      containers.github-runner = inputs.nixpkgs-github-runner.lib.nixosSystem  {
```

Comment:
seems odd to use someone else's lib to get a nixosSystem when you already have it from your nixpkgs import.

In fact, I'm not sure what other import is even for in this case? Maybe it's not utilized fully yet?

Reply:
Hm, I see, you want the other thing's package set to make up the container. Seems a bit weird to me, but ok.

Reply:
the rationale is to use the nixpkgs-github-runner input for the container config. it's different from the upstream nixpkgs

Reply:
Oh... I see, it's probably just a nixpkgs fork with some fixes or something. Got it.